src/api/l_headset.c

```diff
@@ -578,3 +587,93 @@ int luaopen_lovr_headset(lua_State* L) {
   headsetRenderData.ref = LUA_NOREF;
   return 1;
 }
+
+// TEMPORARY HANDS LIBRARY
+
+#include "headset/oculus_mobile_bridge.h"
+#include "headset/oculus_mobile.h"
+
+static void pushTableOfNumbers(lua_State* L, int idx, int count, ...) { // Takes floats
+  va_list args;
+  va_start(args, count);
+
+  lua_newtable(L);
+  for(int c = 0; c < count; c++) {
+    // Note we pass floats but they become doubles because of some garbage about how va_list works.
+    // This is probably okay becuase lua_number is probably a double anyway.
+    lua_pushnumber(L, va_arg(args, double));
+    lua_rawseti (L, -2, c+1);
+  }
+  lua_rawseti (L, -2, idx);
+
+  va_end(args);
+}
+
+static int l_lovrHeadsetHandsIsHand(lua_State* L) {
```

Comment:
True if the hand has hand tracking

Reply:
I think this should be integrated into `isTracked` and/or the return value for `getSkeleton`.

Reply:
Merging this with lovr.headset.isTracked could make sense, like maybe it could return an enum. Should it still be called isTracked at that point?

Should whatever enum we go with be able to distinguish knuckle / oculus touch-level approximate tracking from fully articulated hand tracking?

Reply:
I was just thinking `lovr.headset.isTracked(device, 'skeleton')` since there was talk about doing `lovr.headset.isTracked(device, 'position')` in the past.  It's true that it doesn't expose the "estimated vs. accurate" tracking level.

Reply:
oh, `lovr.headset.isTracked(device, 'skeleton')` is good. I like how that could support several levels of complexity but still degrades to the existing API…